### ISSUE TITLE
CHANGELOG.md

### ISSUE THREAD

**Issue by ghost** [ntaylor1781/k8stk#7]
BLUF: I propose adding change log file and allowing it to trigger automation for releases.

I propose adding a changlog in this format:
https://github.com/majgis/change-log/blob/master/CHANGELOG.md

That repo was an experiment with nodejs cli tool to help creating a standardized changelog based on semver.  I want to eventually rewrite the cli in go, and further explore this idea.

For example, we accumulate entries in Unreleased section of change log, categorized as either major, minor or path.  Then, when ready to release, it is easy to determine semver (and can also have cli calculate it for you with `chlog release`).  Upon merge to master, if latest version is unpublished, it will publish release automatically for you.

I would follow up after this change, with a PR to add automation for releases based on changes to CHANGELOG.md.

**Comment by ntaylor1781** [comment]
Sounds good to me. Do you want to run with that?

**Comment by ghost** [comment]
Yes, if you beat me to adding a change, go ahead and populate it.
Otherwise, I'll add it along with automation to push releases.
